src/core/common/ResourseTemplates/ListTemplate.js

```diff
@@ -139,6 +155,12 @@ class ListTemplate extends Component {
         })
     };
 
+    toggleListBlock = () => {
+        this.setState({
+            isListOpened: !this.state.isListOpened,
```

Comment:
no, use the function form of `setState`:

```js
this.setState(state => ({ isListOpened: !state.isListOpened }));
```

Using `this` in `setState` will cause bugs due to race conditions, as explained in https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous